frontend/components/AlignmentSelector.cpp

```diff
@@ -0,0 +1,323 @@
+/******************************************************************************
+    Copyright (C) 2025 by Taylor Giampaolo <warchamp7@obsproject.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include "AlignmentSelector.hpp"
+
+#include <QPainter>
+#include <QMouseEvent>
+#include <QStyleOptionButton>
+
+#include <QAccessible>
+#include "AccessibleAlignmentSelector.hpp"
+
+#include <util/base.h>
```

Comment:
I might have this slightly incorrect and @PatTheMav will have to correct me:
```suggestion
#include "AccessibleAlignmentSelector.hpp"

#include <util/base.h>

#include <QAccessible>
#include <QMouseEvent>
#include <QPainter>
#include <QStyleOptionButton>
```

Reply:
Nope, that looks correct:

* Working-directory includes first
* First-party includes next
* Third-party includes after